cmake/CompilerHelper.cmake

```diff
@@ -4,6 +4,9 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "PGI" OR CMAKE_CXX_COMPILER_ID MATCHES "NVHPC")
   # CMake adds standard complaint PGI flag "-A" which breaks compilation of of spdlog and fmt
   set(CMAKE_CXX14_STANDARD_COMPILE_OPTION --c++14)
 
+  # Avoid errors related to "excessive recursion at instantiation of function ..."
+  list(APPEND NMODL_EXTRA_CXX_FLAGS "-Wc,--pending_instantiations=0")
```

Comment:
What code needs this, out of curiosity?

Reply:
it is needed for Eigen's transposeInPlace() when compiled with nvhpc, e.g.
https://github.com/BlueBrain/nmodl/blob/kotsalos/eigen_removal/src/solver/newton/newton.hpp#L79